code/game/objects/items/weapons/power_cells.dm

```diff
@@ -313,3 +313,33 @@
 	matter = list(MATERIAL_STEEL = 1)  //some cost just in case you manage to get this in a disk or something
 	maxcharge = 100  //small cause if someone manage to get this shouldn't be that usefull. and 100 is a nice number to work with.
 	spawn_blacklisted = TRUE
+	
+// Infinite cells - intended for debug use only, plus a bit of BSL lore should a player ever get their hands on them. they never run out of charge
+// and have the highest capacity in the game
+
+/obj/item/weapon/cell/large/moebius/nuclear/infinite
+	name = "BSL \"Nullcell 99999L\""
+	desc = "Bluespace League branded rechargeable L-standardized power cell. This strange piece of technology has the tag 'Made in Finland' on the back. It never seems to run out of charge."
+	icon_state = "infinite_b"
+	autorecharge_rate = 1 // charges 100% of itself every tick
+	bad_type = /obj/item/weapon/cell/large/moebius/nuclear/infinite // really shouldn't spawn in maint, or anywhere else
+	maxcharge = 99999 // unlimited power! well, not really, but i dont think anything consumed 99999 watts a tick
+	matter = list(MATERIAL_PLASMA = 12, MATERIAL_STEEL = 3, MATERIAL_PLASTIC = 3, MATERIAL_SILVER = 3, MATERIAL_URANIUM = 12) // might as well give them material values
+
+/obj/item/weapon/cell/medium/moebius/nuclear/infinite
+	name = "BSL \"Nullcell 9999L\""
+	desc = "Bluespace League branded rechargeable M-standardized power cell. This strange piece of technology has the tag 'Made in Finland' on the back. It never seems to run out of charge."
+	icon_state = "infinite_m"
+	autorecharge_rate = 1
+	bad_type = /obj/item/weapon/cell/medium/moebius/nuclear/infinite
+	maxcharge = 9999
+	matter = list(MATERIAL_PLASMA = 8, MATERIAL_STEEL = 2, MATERIAL_PLASTIC = 2, MATERIAL_SILVER = 2, MATERIAL_URANIUM = 8)
+	
+/obj/item/weapon/cell/small/moebius/nuclear/infinite
+	name = "BSL \"Nullcell 999L\""
+	desc = "Bluespace League branded rechargeable S-standardized power cell. This strange piece of technology has the tag 'Made in Finland' on the back. It never seems to run out of charge."
+	icon_state = "infinite_s"
+	autorecharge_rate = 1
+	bad_type = /obj/item/weapon/cell/small/moebius/nuclear/infinite
+	maxcharge = 999
```

Comment:
being that this is a debug item maybe all these power cells should have maxchage = 99999

Reply:
I don't think the number really matters due to 'autorecharge_rate' being 100%. I just had them be seperate numbers to be easier to tell which is which if looking at them in a stack.